openapi.yaml

```diff
@@ -299,8 +299,20 @@ definitions:
           additionalProperties: true
         integratedTime:
           type: integer
-        inclusionProof:
-          $ref: '#/definitions/InclusionProof'
+        verification:
+          type: object
+          properties:
+            inclusionProof:
+              $ref: '#/definitions/InclusionProof'
+            signedEntryTimestamp:
+              type: string
+              format: byte
+              # To verify the signedEntryTimestamp:
```

Comment:
if you end up making another change to this, I might add change the wording to be a bit less Golang specific - for example:

- Remove the verification object from the JSON document
- Follow RFC 8785 rules to canonicalize the remaining JSON document

Reply:
good call, added that in